specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/securitydomain.json

```diff
@@ -226,7 +243,8 @@
     "SecurityDomainObject": {
       "properties": {
         "value": {
-          "type": "string"
+          "type": "string",
+          "description": "Security domain"
```

Comment:
You should add `"format": "base64"` (or "base64url" for consistency with rest of KV, but either is fine) to clients know how to encode the data, if this is indeed the encoded data we discussed in our meeting.